src/modules/Cluster.rb

```diff
@@ -349,7 +349,7 @@ def SaveClusterConfig
       end
 
       # BNC#883235. Enable "two_node" when using two node cluster
-      if (@expected_votes == "2") or (@memberaddr.size == 2)
+      if (@expected_votes == "2") or (@memberaddr.size == 2) or (!@corosync_qdevice)
```

Comment:
The logic is incorrect, should change to `(!@corosync_qdevice) and (@expected_votes == "2" or @memberaddr.size == 2)`